manifests/pulp_client.pp

```diff
@@ -43,5 +43,14 @@
       mode  => '0440',
     }
   }
-}
 
+  foreman_config_entry { 'pulp_client_cert':
+    value => $client_cert,
+    ignore_missing => false
```

Comment:
We prefer trailing comma's because it gives a smaller diff when we add additional parameters.